src/main/java/org/apache/commons/compress/archivers/sevenz/AES256Options.java

```diff
@@ -0,0 +1,34 @@
+/*
+ *  Licensed to the Apache Software Foundation (ASF) under one or more
+ *  contributor license agreements.  See the NOTICE file distributed with
+ *  this work for additional information regarding copyright ownership.
+ *  The ASF licenses this file to You under the Apache License, Version 2.0
+ *  (the "License"); you may not use this file except in compliance with
+ *  the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+package org.apache.commons.compress.archivers.sevenz;
+
+import java.util.Arrays;
+import java.util.Random;
+
+public class AES256Options {
+    byte[] password;
+    byte[] salt = new byte[0];
+    byte[] iv = new byte[16];
+    int numCyclesPower = 19;
+
+    public AES256Options(byte[] password) {
+        this.password = password;
+        new Random(Arrays.hashCode(password)).nextBytes(salt);
+        new Random(Arrays.hashCode(password)).nextBytes(iv);
```

Comment:
Do we need to worry about the `Random` instead of `SecureRandom` or some other random class from Commons Rng/etc?

Reply:
Good suggestion, as `SecureRandom` doc says : 
> This class provides a cryptographically strong random number generator (RNG)

👉🏼 So I will switch to it